plugins/config-file.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-file               
+spec:
+  version: "0.0.6"      
+  platforms:
+  - selector:             
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/Modulus/kubectl-cfg/archive/0.0.6.zip
+    sha256: "41952b7105e2a9f5f1b3fec306bc59c8e972578f9b5d543defeb970c5b5876d5"
+    files:
+    - from: "/kubectl-config-file*/*.sh" 
+      to: "."              
+    bin: "./kubectl-config-file.sh"  
+  shortDescription: List config files, and links chosen config file to ~/.kube/config without messing up your files.
+  homepage: https://github.com/Modulus/kubectl-cfg 
+  caveats: |
+    This plugin needs the following programs:
+    * ls(1)
+    * ln(2)
+    * find(3)
+    * grep(4)
+    * cut(5)
+    * echo(6)
+  description: |
+    This plugin list all files in ~/.kube folder. Can also link chosen file as ~/.kube/config. So you can work with multiple cluster configs without messing with environment variables.
```

Comment:
please line wrap this to 80 chars.
also off-topic, I think people `rm -rf ~/.kube` all the time, I recommend finding a better path you can recommend people to store their kubeconfigs.

Reply:
I have wrapped this line to 80 chars.

Reply:
Can you push the fix?

Reply:
Sorry. Done :)